languages/go/bitwarden_client.go

```diff
@@ -0,0 +1,67 @@
+package sdk
+
+import (
+	"encoding/json"
+
+	"github.com/bitwarden/sdk/languages/go/internal/cinterface"
+)
+
+type BitwardenClient struct {
+	client        cinterface.ClientPointer
+	lib           cinterface.BitwardenLibrary
+	commandRunner CommandRunnerInterface
+	Projects      ProjectsInterface
+	Secrets       SecretsInterface
+}
+
+func NewBitwardenClient(apiURL *string, identityURL *string) (*BitwardenClient, error) {
+	deviceType := DeviceType("SDK")
+	var userAgent = "GOLANG-SDK"
+	clientSettings := ClientSettings{
+		APIURL:      apiURL,
+		IdentityURL: identityURL,
+		UserAgent:   &userAgent,
+		DeviceType:  &deviceType,
+	}
+
+	settingsJSON, err := json.Marshal(clientSettings)
+	if err != nil {
+		return nil, err
+	}
+
+	lib := cinterface.NewBitwardenLibrary()
+	client, err := lib.Init(string(settingsJSON))
+	if err != nil {
+		return nil, err
+	}
+	runner := NewCommandRunner(client, lib)
+
+	return &BitwardenClient{
+		lib:           lib,
+		client:        client,
+		commandRunner: runner,
+		Projects:      NewProjects(runner),
+		Secrets:       NewSecrets(runner),
+	}, nil
+}
+
+func (c *BitwardenClient) AccessTokenLogin(accessToken string) (*ResponseForAPIKeyLoginResponse, error) {
```

Comment:
We should change this to only return an error if one exists, since the consumer has no use for the response.

```suggestion
func (c *BitwardenClient) AccessTokenLogin(accessToken string) (error) {
```

This also means checking that the response is successful. Similar to the .NET implementation, https://github.com/bitwarden/sdk/blob/master/languages/csharp/Bitwarden.Sdk/BitwardenClient.cs#L23-L31

Reply:
Implemented to return just error